addon/components/bs-button.d.ts

```diff
@@ -0,0 +1,54 @@
+import Component from '@glimmer/component';
+
+export type BsButtonType = 'primary' | 'secondary' | 'success' | 'danger' | 'warning' | 'info' | 'light' | 'dark';
+
+interface BsButtonSignature {
+  Element: HTMLButtonElement;
+  Args: {
+    active?: boolean;
+    block?: boolean;
+    bubble?: boolean;
+    buttonType?: 'button' | 'submit';
+    defaultText?: string;
+    fulfilledText?: string;
+    icon?: string;
+    iconActive?: string;
+    iconInactive?: string;
+    outline?: boolean;
+    pendingText?: string;
+    preventConcurrency?: boolean;
+    rejectedText?: string;
+    reset?: boolean;
+    size?: string;
+    type?: BsButtonType;
+    value?: unknown;
+    onClick?: (e?: Event, value?: unknown) => void;
+  };
+  Blocks: {
+    default: [
+      {
+        isFulfilled: boolean;
+        isPending: boolean;
+        isRejected: boolean;
+        isSettled: boolean;
+        actions: BsButtonActions;
```

Comment:
Where is this coming from? I don't see it is in yielded hash: https://github.com/ember-bootstrap/ember-bootstrap/blob/4ec513d3dd54e0d6d91a64dcb32cf3b57989bf93/addon/components/bs-button.hbs#L9-L16

Reply:
if you mean `actions`, I was under impression this is how you should type them for Glint, but I might be wrong. will check later.
The others are in the yielded hash.